src/store/CurrentAccount/actions.ts

```diff
@@ -0,0 +1,41 @@
+import {CurrentAccountInterface} from "@/interfaces";
+import {SessionStorageKeys} from "sbc-common-components/src/util/constants";
+import {sleep} from "@/utils";
+import currentAccount from "@/store/CurrentAccount/index";
+
+export default {
+  async loadCurrentAccount (context): Promise<any> {
+    return new Promise((resolve, reject) => {
+      context.dispatch('waitForCurrentAccount')
+        .then((currentAccount) => {
+          const accountInfo: CurrentAccountInterface = {
+            accountType: currentAccount.accountType,
+            id: currentAccount.id,
+            label: currentAccount.label,
+            type: currentAccount.type
+          }
+          context.commit('setAccountInformation', accountInfo)
+          resolve()
+        })
+        .catch(() => {
+          reject()
+        })
+    })
+  },
+
+  async waitForCurrentAccount (): Promise<any> {
```

Comment:
Please copy the comments here as well (or write your own). Mainly, why are we trying 50 times?

Reply:
Good idea, I'll do it.